From 28fa779ecad7642281b9327356ebb51b2fc97d85 Mon Sep 17 00:00:00 2001
From: Harsha Babu Tumkur Anand Shankar Babu <>
Date: Fri, 2 Aug 2024 17:47:10 +0000
Subject: [PATCH] Remove menu entries for merge conflict and call items

This patch remove the context menu entries for merge conflict options and merge conflict navigation, call items in the references view.
---
 extensions/merge-conflict/package.json        | 26 -------
 extensions/references-view/package.json       | 10 ---
 .../parts/activitybar/activitybarPart.ts      |  2 +-
 .../browser/parts/globalCompositeBar.ts       | 68 +++++++++----------
 4 files changed, 35 insertions(+), 71 deletions(-)

diff --git a/extensions/merge-conflict/package.json b/extensions/merge-conflict/package.json
index 737f842..b7ebc08 100644
--- a/extensions/merge-conflict/package.json
+++ b/extensions/merge-conflict/package.json
@@ -29,32 +29,6 @@
     "watch": "gulp watch-extension:merge-conflict"
   },
   "contributes": {
-    "menus": {
-      "scm/resourceState/context": [
-        {
-          "command": "merge-conflict.accept.all-current",
-          "when": "scmProvider == git && scmResourceGroup == merge",
-          "group": "1_modification"
-        },
-        {
-          "command": "merge-conflict.accept.all-incoming",
-          "when": "scmProvider == git && scmResourceGroup == merge",
-          "group": "1_modification"
-        }
-      ],
-      "editor/title": [
-        {
-          "command": "merge-conflict.previous",
-          "group": "navigation@1",
-          "when": "!isMergeEditor && mergeConflictsCount && mergeConflictsCount != 0"
-        },
-        {
-          "command": "merge-conflict.next",
-          "group": "navigation@2",
-          "when": "!isMergeEditor && mergeConflictsCount && mergeConflictsCount != 0"
-        }
-      ]
-    },
     "configuration": {
       "title": "%config.title%",
       "properties": {
diff --git a/extensions/references-view/package.json b/extensions/references-view/package.json
index 3952cb6..fa92c55 100644
--- a/extensions/references-view/package.json
+++ b/extensions/references-view/package.json
@@ -259,16 +259,6 @@
           "group": "2@3",
           "when": "view == references-view.tree && viewItem == file-item || view == references-view.tree && viewItem == reference-item"
         },
-        {
-          "command": "references-view.showOutgoingCalls",
-          "group": "1",
-          "when": "view == references-view.tree && viewItem == call-item"
-        },
-        {
-          "command": "references-view.showIncomingCalls",
-          "group": "1",
-          "when": "view == references-view.tree && viewItem == call-item"
-        },
         {
           "command": "references-view.showSupertypes",
           "group": "1",
diff --git a/src/vs/workbench/browser/parts/activitybar/activitybarPart.ts b/src/vs/workbench/browser/parts/activitybar/activitybarPart.ts
index 15e58a5..f4ae927 100644
--- a/src/vs/workbench/browser/parts/activitybar/activitybarPart.ts
+++ b/src/vs/workbench/browser/parts/activitybar/activitybarPart.ts
@@ -251,7 +251,7 @@ export class ActivityBarCompositeBar extends PaneCompositeBar {
 		// Global Composite Bar
 		if (this.globalCompositeBar) {
 			actions.push(new Separator());
-			actions.push(...this.globalCompositeBar.getContextMenuActions());
+			// actions.push(...this.globalCompositeBar.getContextMenuActions());
 		}
 		actions.push(new Separator());
 		actions.push(...this.getActivityBarContextMenuActions());
-- 
2.40.1

